### PR TITLE
Change docs and ipynb to reflect new import path for UnityEnvironment

### PR DESCRIPTION
**Pull request by ervteng**
Looks like my editor also removed a bunch of white spaces.

### CODE REVIEW

File: docs/Migrating.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 * `UnitySDK/Assets/ML-Agents/Scripts/Communicator.cs` and its class `Communicator` have been renamed to `UnitySDK/Assets/ML-Agents/Scripts/ICommunicator.cs` and `ICommunicator` respectively.
 * The `SpaceType` Enums `discrete`, and `continuous` have been renamed to `Discrete` and `Continuous`.
 * We have removed the `Done` call as well as the capacity to set `Max Steps` on the Academy. Therefore an AcademyReset will never be triggered from C# (only from Python). If you want to reset the simulation after a fixed number of steps, or when an event in the simulation occurs, we recommend looking at our multi-agent example environments (such as BananaCollector). In our examples, groups of Agents can be reset through an "Area" that can reset groups of Agents.
+* The import for `UnityEnvironment` in the low-level Python API has changed. If you are using the Python API, change `from mlagents.envs import UnityEnvironment` to `from mlagents.envs.environment import UnityEnvironment`.
```
Comment (chriselion) [inline docs/Migrating.md]: This isn't quite accurate. UnityEnvironment is still in ml-agents-envs/mlagents/envs/environment.py, but an import that made it look like it was also in another location was removed.

Comment (chriselion) [reply docs/Migrating.md]: Here's the actual line that changed:
https://github.com/Unity-Technologies/ml-agents/pull/2584/files#diff-829b0b865c5a9e2ea85ed647b3012702L4

Comment (ervteng) [reply docs/Migrating.md]: Ah, I see - should it read something like "The import for mlagents.envs.UnityEnvironment was removed"?

Comment (chriselion) [reply docs/Migrating.md]: Sounds reasonable

**Pull request merged**